bundles/org.openhab.ui/web/cordova/config.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version='1.0' encoding='utf-8'?>
+<widget id="org.openhab.ui" version="3.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+    <name>openHAB</name>
+    <description>
+        A sample Apache Cordova application that responds to the deviceready event.
```

Comment:
Err, no.

Reply:
Good catch - I left the Cordova stuff even if we're not sure yet whether we would publish Cordova builds in app stores and so on - we'll discuss it when the time comes, but definitely could, I regularly test iOS and Android builds, there's a additional login screen but overall it's a working full-screen version of the web app without the browser UI. Adding to the home screen from Safari or Chrome works too.